I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 16, in script
    init -1 python:
  File "game/script.rpy", line 22, in <module>
    AjouterEvtsRejMetier()
  File "game/extremis/socio_eco/metiers/evts_metiers.rpy", line 34, in AjouterEvtsRejMetier
    probaMusicien.ajouterModifProbaViaVals(0.05, estConquistaror)
NameError: global name 'estConquistaror' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 16, in script
    init -1 python:
  File "D:\Dev\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Dev\Renpy\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 22, in <module>
    AjouterEvtsRejMetier()
  File "game/extremis/socio_eco/metiers/evts_metiers.rpy", line 34, in AjouterEvtsRejMetier
    probaMusicien.ajouterModifProbaViaVals(0.05, estConquistaror)
NameError: global name 'estConquistaror' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
 
Sun Feb 21 22:21:40 2021
